mix.exs

```diff
@@ -36,10 +36,10 @@ defmodule Numbers.Mixfile do
     [
       {:coerce, "~> 1.0"},
 
-      {:decimal, "~> 1.9", optional: true},
+      {:decimal, "~> 2.0", optional: true},
```

Comment:
Could we change it to the more inclusive `~> 1.9 or ~> 2.0`? I'm pretty sure that for whatever reason there are packages which for other reasons are stuck on an older version of Decimal.

AFAICS none of the backwards-incompatible changes in Decimal 2.0 affect Numbers, at least :-).